gulpfile.js

```diff
@@ -2,10 +2,43 @@
 
 'use strict';
 
+const { spawn } = require('child_process');
 const fs = require('fs');
 const gulp = require('gulp');
 const sass = require('sass');
 
+/**
+ * Task to output a draft changelog to be appended to CHANGELOG.md at the
+ * top of the file. Remove git logs that are not relevant for an consumer
+ * and be sure to emphasize features are breaking with "BREAKING CHANGE:"
+ *
+ * See docs/releases.md
+ */
+gulp.task('changelog', async done => {
+  spawn('git', ['describe', '--abbrev=0', '--tags'], {
```

Comment:
We could use `run` here, but it would need to be modified because the current version in the client resolves on "exit" not on "data". I just did the super stupid thing and nested the spawns. -- for better or worse.